### ISSUE TITLE
Ask QA for a QA plan outline

### ISSUE THREAD

**Issue by daviddahl** [Crypton/statusapp#56]
I'll work something up, but it might be nice to follow a pre-set recepie

**Comment by daviddahl** [comment]
@reneelynn do you have a preferred format?

**Comment by reneelynn** [comment]
@devgeeks 's model should work fine if the following sounds good to you: email me with the link to the build and I'll write up a card for Trello. I'll then run tests in TestLodge and ticket bugs here. At the end, I'll reply to your email with the listed relevant bugs for this time around.

I will need documentation on expected behaviors and branding so I can write up TestLodge tests. Also please be advised that with 6.0 on the near horizon this will not be top priority.

**Comment by daviddahl** [comment]
hrmm, any way we can not involve Trello? Less Hellhole more Github :)

**Comment by daviddahl** [comment]
also, what is @devgeeks model?

**Comment by reneelynn** [comment]
That ^^ is how @devgeeks and I handle mobile builds.

Trello doesn't really involve anyone except me and my team. It's where we keep our priority tracking. Without it going on there I have to answer more questions about priority than I want to spend my time doing.

**Comment by daviddahl** [comment]
Here is the current QA plan, feel free to add new (better) ideas here and I can update the Gist: https://gist.github.com/daviddahl/0fe8a04b98724c7583f8

**Comment by daviddahl** [comment]
moving to backlog, but is fixed for now:)

**Comment by reneelynn** [comment]
I have everything in TestLodge now so you are free to close this if you prefer. I'll be updating tests as Kloak gets new functionality.

**Comment by daviddahl** [comment]
Yeah, just keeping it around in the backlog to remind me to also do the same
